smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -286,4 +281,15 @@ class SwiftWriter(private val fullPackageName: String) : CodeWriter() {
         function()
         dedent()
     }
+
+    init {
+        trimBlankLines()
+        trimTrailingSpaces()
+        setIndentText("    ")
+        putFormatter('D', SwiftSymbolFormatter(shouldSetDefault = true, shouldRenderOptional = true))
+        putFormatter('T', SwiftSymbolFormatter(shouldSetDefault = false, shouldRenderOptional = true))
+        putFormatter('N', SwiftSymbolFormatter(shouldSetDefault = false, shouldRenderOptional = false))
```

Comment:
What are the usecases for these formatters? Could comments be added here for quick reference?